core/modules/installer/installer.manager.inc

```diff
@@ -298,13 +295,7 @@ function installer_manager_update_form($form, $form_state) {
  * @see installer_manager_update_form_submit()
  */
 function installer_manager_update_form_validate($form, &$form_state) {
-  if (!empty($form_state['values']['projects'])) {
-    $enabled = array_filter($form_state['values']['projects']);
-  }
-  if (!empty($form_state['values']['disabled_projects'])) {
-    $disabled = array_filter($form_state['values']['disabled_projects']);
-  }
-  if (empty($enabled) && empty($disabled)) {
+  if (empty(array_filter($form_state['values']['projects']))) {
```

Comment:
Before php 5.5 empty() only takes *variables* as parameter. Try to move that array_filter() out of it.

Reply:
So obvious 🤦🏼 ...thank you @indigoxela ❤️